components/dialog/action-buttons.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Action Buttons
+page_title: Dialog Action Buttons
+description: Action Buttons of the Dialog for Blazor.
+slug: dialog-action-buttons
+tags: telerik,blazor,dialog,action,buttons
+published: True
+position: 7
+---
+
+# Dialog Action Buttons
+
+The Dialog provides options for rendering action buttons and customizing their content. To specify action buttons in the Dialog, use the `DialogButtons` tag. You can customize the layout of the buttons by using the `ButtonsLayout` property.
```

Comment:
add a sentence that explains what you can do with these action buttons, where they are, what benefit you get by reading this article, what can yo udo now that you could not do before